tests/app.test.js

```diff
@@ -150,7 +150,7 @@ describe('Various ways to open and close InVEST models', () => {
       findByText,
       findByTitle,
       findByRole,
-      findAllByText,
+      findAllByText
```

Comment:
FYI, eslint suggests using the trailing commas. I don't feel strongly, but if you see them, they're not there by accident!

Reply:
Oh, good to know! I'll put it back.